content/docs/user-guide/packaging/project-export-cli.md

```diff
@@ -0,0 +1,271 @@
+{{< /note >}}
+
+## Prerequisites
+To make best use of the `export-project` command, it is recommended to have a project with at least one starting scene, and all necessary seedlist files prepared. AssetProcessor Registry file settings may need to be tweaked. To learn how to set this up, please consult the following page: [Creating a Project Game Release Layout for Windows, section: Set the Starting Level](windows-release-builds/#set-the-starting-level).
+
+To learn more about the AssetBundler and Seed Files, please visit the [overview page on the AssetBundler tool](https://docs.o3de.org/docs/user-guide/packaging/asset-bundler/overview/).
+
+## Getting Started
+To use the `export-project` command correctly, it is important to first understand how it works at a high level.
+
+The command operates in 2 stages:
+1. Script Loading
+2. Script Running
+
+### Script Loading
+Before the export process can begin, the export-project command does the following pre-processing steps:
+* Parse and validate initial set of arguments, and filters arguments to be processed by desired export script
+* Load the export script as a python module
+* Validate the expected project path
+* Modify the system path with O3DE CLI Python API so that user scripts can easily import API code and modules
+* Prepares the O3DE script context, which contains useful values like project path, engine path, project name, CMake args, etc.
+* Runs the export script with the prepared context injected
+
+### Script Running
+The `export-project` command can run any python script that the user designates as the export script. As a result, the exact behavior entirely depends on the supplied script. O3DE includes a standard script, which is described [further below.](#standard-export-script)
+
+Any script run by the command has access to the running context of the exporter, along with common APIs.
+
+## Export CLI Command
+The `export-project` command is accessed via the O3DE CLI that is shipped with the engine. This is found in `<engine>/scripts/o3de.bat`, or on Unix systems as `<engine>/scripts/o3de.sh`. The command has the following arguments:
+
+| Argument Name | Description | Required? |
+| - | - | - |
+| `--export-script` | Choose an external python script to run. The path can be relative or absolute. | yes |
+| `--project-path` | Project to export. The path can be relative or absolute. If not supplied, it will be inferred by the export script location. | no |
+| `--log-level` | Sets the verbosity of the logs emitted from the exporter. INFO means all information will be emitted. ERROR means it will  be silent unless a failure occurs. Choices are `[DEBUG, INFO, WARNING, ERROR, CRITICAL]`, with `ERROR` as the default. | no |
+
+You can inspect the code for parsing arguments [here.](https://github.com/o3de/o3de/blob/753480ec930e55f3431f92ed7b974ba7f9e73a13/scripts/o3de/o3de/export_project.py#L354)
```

Comment:
Throughout here in the links to specific files, instead of using the commit has you can use `development` so it will always just take the user to the latest version of that file in the `development` branch.
```suggestion
You can inspect the code for parsing arguments [here.](https://github.com/o3de/o3de/blob/development/scripts/o3de/o3de/export_project.py#L354)
```